### ISSUE TITLE
Don't use @container @set for "label"

### ISSUE THREAD

**Issue by acka47** [hbz/lobid-resources#808]
This is strange. We have this in the [lobid-resources JSON-LD context](http://lobid.org/resources/context.jsonld):

```json
{
  "label": {
    "@id": "http://www.w3.org/2000/01/rdf-schema#label",
    "@container": "@set"
  }
}
```

but the `label` usually has no array but a string as value. Trying this context out int he JSON-LD playground all label statements get an array. I think we should adjust the context accordingly.

**Comment by ChristophEwertowski** [comment]
Yes `label` should be strings. At least in the API description pages there is no label which is an array. I removed it and referred the pull request to dr0i.

**Comment by dr0i** [comment]
<tl;dr>All is well</tl;dr>
At the scrum @ChristophEwertowski and @fsteeg and me discussed this issue.
In the tests files we had in fact labels as arrays for the `items` because the json-comparator didn't recognize this as a difference. Setting the context explicitly to be no `set` the test files were rebuild at some point (strangely not part in the PR #810) but at some point of the merge process I exeperienced at our staging deployment.
However  - nothing is changed in our cluster data, _only_ item-test-files are affected, so:
all is well and good.
